test/unittests/mongoBackend/mongoUpdateContextSubscription_test.cpp

```diff
@@ -561,7 +561,7 @@ TEST(mongoUpdateContextSubscription, updateDuration)
 
     EXPECT_EQ(60, sub.getIntField("throttling"));
     EXPECT_STREQ("http://notify1.me", C_STR_FIELD(sub, "reference"));
-    EXPECT_STREQ("JSON", C_STR_FIELD(sub, "format"));
+    EXPECT_STREQ("legacy", C_STR_FIELD(sub, "format"));
```

Comment:
Hmmm, I did a grep on "legacy" and after the changes, all unit tests worked.
I will look into it (in all places, no need to add more comment about the same in the PR)